src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -354,7 +354,6 @@ class NBDServer
     void* entry() override
     {
       (server.*func)();
-      server.shutdown();
```

Comment:
Sure. Or you can create a new PR for "remove debug messages" fix (and remove this commit from here).